modules/sor/sor.resolvers.ts

```diff
@@ -17,13 +16,8 @@ const balancerSdkResolvers: Resolvers = {
 
             return sorService.getSorSwaps(args);
         },
-        sorGetBatchSwapForTokensIn: async (parent, args, context) => {
-            const tokens = await tokenService.getTokens();
-
-            return balancerSorService.getBatchSwapForTokensIn({ ...args, tokens });
-        },
-        sorGetCowSwaps: async (parent, args, context) => {
-            return sorService.getCowSwaps(args);
+        sorV2GetSwaps: async (parent, args, context) => {
+            return sorService.getSorV2Swaps(args);
```

Comment:
are we going to have a different one for V3? I wonder if we shouldn't aim for unified SOR. Would we have a router that could support chained swaps on both vaults @danielmkm wdyt?